src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/forgotpassword/AttachedForgotPasswordPage.java

```diff
@@ -0,0 +1,56 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth.forgotpassword;
+
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.AuthPageContext;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormError;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.FormPage;
+import lombok.Getter;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AttachedForgotPasswordPage extends BasePageObject implements ForgotPasswordPage {
+
+  @FindBy(css = "#forgotPasswordSubmit")
+  private WebElement requestLinkButton;
+  @FindBy(css = "#loginUsername")
+  private WebElement usernameField;
+
+  private final String PASS_REQUESTED_SUCCESS = "Thanks! Please check your email.";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1170)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "PASS_REQUESTED_SUCCESS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS00116)